narratives/ncov_sit-rep_2020-03-02.md

```diff
@@ -299,6 +297,87 @@ This estimates are broadly consistent with those by other scientists which mostl
 Importantly, R0 is a quantity that depends strongly on the socio-economic context and infection control measures.
 ```
 
+# [The Nextstrain team's take on COVID-19 frequent questions](https://nextstrain.org/ncov/2020-01-30/2020-01-30)
+
+As a team, we’ve been fielding a lot of questions about COVID-19 and how individuals and families should be preparing. We have compiled a list of the most common questions we receive and answered them with information and recommendations from resources that we trust. 
+
+With all of this, it is important to note that as COVID-19 is a novel virus, we are still learning about disease characteristics including who is most likely to become infected or experience serious illness as a result of it. Current clinical knowledge is based primarily on data from the China outbreak. We believe everyone should be taking steps to protect themselves and their families. We hope the resources below will help you to do that. 
+
+```auspiceMainDisplayMarkdown
+## COVID-19 FAQs
+
+#### How quickly is COVID-19 spreading?
+Based on modeling estimates and phylodynamic analysis, the current epidemic doubling time is about 7 days. There is uncertainty with these estimates and this rate will vary based on community contact patterns. Nonpharmaceutical interventions can be effective in reducing this rate, as we have [seen in China](https://twitter.com/jodigraphics15/status/1232484344872071168?s=20). Sudden jumps in case counts are likely due to increased testing capacity catching cases that were not previously missed by surveillance systems.
+
+Current case counts, deaths, and recoveries are being tracked in a number of places. These are updated as regularly as possible, but may not always be immediately up to date. One dashboards that we particularly like is from the JohnsCenter for Systems Science and Engineering:
+[Johns Hopkins CSSE Dashboard](https://gisanddata.maps.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6)
+
+#### How is COVID-19 spread?
+COVID-19 is a respiratory virus, which spread in 3 main ways:
+1. Between people who are in close contact with each other (within about 6 feet, or 2 metres), through small aerosolized particles that can be produced when a person talks, coughs, or sneezes. 
+2. In (relatively) larger droplets that are produced someone coughs or sneezes (these can travel over similar distances)
+3. Contact with surfaces or objects infected with the virus (also known as fomites). If a person touches an infected surface and then touches their eyes, nose, or mouth without washing their hands they may become sick by introducing the virus into their body.
```

Comment:
Link/Source missing if possible

Reply:
I *struggled* to find a good source that summed it up in one location, but I think we could link here: https://www.aappublications.org/content/35/1/1.2

reputable source and does go over all of them.